tmpl/blog/introducing-charrua-dhcp.md

```diff
@@ -0,0 +1,221 @@
+Almost every network needs to support [DHCP]
+(https://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol) (Dynamic
+Host Configuration Protocol), that is, a way for clients to request network
+parameters from the environment, common parameters are an IP address, a network
```

Comment:
Suggest starting a new sentence here: "... from the environment. Common parameters are ..."